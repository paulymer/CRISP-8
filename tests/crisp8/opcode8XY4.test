` a4bÿcþ€‚4