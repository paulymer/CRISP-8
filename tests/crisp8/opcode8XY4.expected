PC: 0x0111 I: 0
V0: 84  V1: 52  V2: 253  V3: 254  V4: 0  V5: 0  V6: 0  V7: 0
V8: 0  V9: 0  Va: 0  Vb: 0  Vc: 0  Vd: 0  Ve: 0  Vf: 1
0200: 6020 6134 62ff 63fe 8014 8234 1111 0000